src/caffe/proto/caffe.proto

```diff
@@ -181,6 +183,8 @@ message SolverParameter {
   optional int32 stepsize = 13;
   // the stepsize for learning rate policy "multistep"
   repeated int32 stepvalue = 34;
+  // the stepsize for learning rate policy "plateau"
+  repeated int32 plateau_winsize = 41;
```

Comment:
Why not just re-use `stepsize` and/or `stepvalue`?

Also, did you mean to use `plateau_winsize` here? Your PR body mentions `plateau_stepsize`.